### DISCUSSION TITLE
Need help moving styles after reinstalling Windows

### DISCUSSION THREAD

**Discussion by Andryshik345** [stylus/stylus#2737] [Q&A] [answered]
Hello, i need somehow transfer all my styles from `windows.old` folder that contains Firefox and Edge settings and profiles. I totally forgot about connecting sync with cloud storages and i also forgot to backup it locally. I had a lot of styles that were written by me and i didn't have any backups anywhere (silly me).
So, is there any way to transfer them to the new Firefox (and maybe edge) install? Simply copying profile from there didn't work. Also tried to search through extension data and settings without any luck.

**Comment by iChenLei** [comment, marked as answer]
@Andryshik345  goto https://github.com/openstyles/stylus

**Reply by Andryshik345** [reply]
oops, my bad, thanks
